presto-native-execution/presto_cpp/main/runtime-metrics/PrometheusStatsReporter.cpp

```diff
@@ -0,0 +1,234 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "PrometheusStatsReporter.h"
+
+#include <prometheus/collectable.h>
+#include <prometheus/counter.h>
+#include <prometheus/gauge.h>
+#include <prometheus/histogram.h>
+#include <prometheus/registry.h>
+#include <prometheus/summary.h>
+#include <prometheus/text_serializer.h>
+
+namespace facebook::presto::prometheus {
+
+// Initialize singleton for the reporter
+folly::Singleton<facebook::velox::BaseStatsReporter> reporter(
+    []() -> facebook::velox::BaseStatsReporter* {
+      return facebook::presto::prometheus::PrometheusStatsReporter::
+          createPrometheusReporter()
+              .release();
+    });
+
+using namespace ::prometheus;
```

Comment:
Can we remove this and use `::prometheus::` where required? It is easier to read the code this way.